index.rst

```diff
@@ -120,6 +115,13 @@ public transportation or a taxi there.
    Pacific Northwest tends to be muddy or raining with occasional swaths
    of blue skies. Layering is usually the way to go.
 
+**What's the weather like?**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Warmed by Pacific Ocean currents and protected by a range of mountains, Vancouver's weather is mild year-round. 
```

Comment:
Remove this section, or write up your own copy.

Reply:
Done